packages/blade/src/components/Badge/Badge.stories.tsx

```diff
@@ -50,7 +51,21 @@ const Page = (): ReactElement => {
       <br />
       <br />
       <Title>Usage</Title>
-      <Highlight language="tsx">{`import { Badge } from '@razorpay/blade/components'; \nimport type { BadgeProps } from '@razorpay/blade/components';`}</Highlight>
+      <Sandbox>
+        {`
+        import { Badge, InfoIcon } from '@razorpay/blade/components';
+        
+        function App() {
```

Comment:
I miss TS here 😭

Reply:
Yesss 😢 . We can add TS but autocomplete won't come in editor :(